test/test_svd.cc

```diff
@@ -138,15 +145,16 @@ void test_svd_work( Params& params, bool run )
         Acpy = slate::Matrix<scalar_t>::fromScaLAPACK(
                 m, n, &Acpy_data[0],  lldA,  nb, p, q, MPI_COMM_WORLD);
 
+        // todo: size m-by-m, n-by-n for AllVec
```

Comment:
Why is this a TODO?  It looks like U is m-by-m and VT is n-by-n when the respective job is Allvec

Reply:
I guess out-dated now.

Reply:
Removed.